includes/class-give-db-donors.php

```diff
@@ -704,4 +705,39 @@ private function bc_200_params() {
 			$wpdb->donors = $this->table_name = "{$wpdb->prefix}give_customers";
 		}
 	}
+
+	/**
+	 * Update Donor Information when User Profile is updated from admin.
+	 *
+	 * @param int $user_id
+	 *
+	 * @access public
+	 * @since  2.0
+	 *
+	 * @return bool
+	 */
+	public function update_donor_info_on_user_update( $user_id = 0 ) {
+
+		if ( current_user_can('edit_user',$user_id) ) {
+
+			$donor = new Give_Donor( $user_id, true );
+
+			// Bailout, if donor doesn't exists.
+			if ( ! $donor ) {
+				return false;
+			}
+
+			// Get User First name and Last name.
+			$first_name = ( $_POST['first_name'] ) ? $_POST['first_name'] : get_user_meta( $user_id, 'first_name', true );
+			$last_name  = ( $_POST['last_name'] ) ? $_POST['last_name'] : get_user_meta( $user_id, 'last_name', true );
+			$full_name  = strip_tags( stripslashes( $first_name . ' ' . $last_name ) );
```

Comment:
@mehul0810 use `wp_unslash` as you did before in same PR and use trim function with double quote string